src/website/views/components/forms/search-input/_examples.pug

```diff
@@ -390,5 +475,22 @@ script.
     'DOMContentLoaded',
     function() {
       chi.tab(document.querySelectorAll('.chi-tabs-panel .chi-tabs'));
+
+      const input = document.getElementById("with-dropdown");
+
+      input.addEventListener("chiInput", () => {
+        document.getElementById("dropdown").show(); // toggle()
+      })
+
+      input.addEventListener("chiBlur", () => {
+        document.getElementById("dropdown").hide(); // hide()
+      })
     }
   );
+
+
+style.
+  #dropdown .chi-dropdown__menu.-z--10.sc-chi-dropdown.sc-chi-dropdown-s.-active {
+    position: absolute;
+  }
+
```

Comment:
```suggestion


```

Reply:
@lumdmtr I have removed in codepen file and it worked fine. I think is not neccessary